blint/lib/utils.py

```diff
@@ -223,21 +226,29 @@ def blintdb_setup(args):
     and puts it into $BLINTDB_LOC path.
     If there is not path in $BLINTDB_LOC, it will add it to $HOME/blindb.
     $USE_BLINTDB is required to be set "true" or "1", in order to use blintdb
-    """
-    if not os.getenv("USE_BLINTDB") or not args.use_blintdb :
+    """    
+    if not os.getenv("USE_BLINTDB") and not args.use_blintdb :
         LOG.debug(f"Skipping blintdb setup, USE_BLINTDB={os.getenv('USE_BLINTDB')}")
         return
 
     if not os.path.exists(BLINTDB_HOME):
         os.makedirs(BLINTDB_HOME)
-
-    oras_client = oras.client.OrasClient()
-    oras_client.pull(
-        target="ghcr.io/appthreat/blintdb-vcpkg:v0.1",
-        outdir=BLINTDB_HOME,
-        allowed_media_type=[],
-        overwrite=False,
-    )
+    try:
+        oras_client = oras.client.OrasClient()
+        oras_client.pull(
+            target="ghcr.io/appthreat/blintdb-vcpkg:v0.1",
+            outdir=BLINTDB_HOME,
+            allowed_media_type=[],
+            overwrite=False,
```

Comment:
@aryan-rajoria I'm still not happy with requesting a download every time blint gets run with blintdb enabled and not giving users any way to override that behavior. What would you think of making either a flag or an environment variable to opt-in to "refresh" the blintdb rather than making that the default?

Reply:
yes, that makes sense

Reply:
Have added a new environment variable `BLINTDB_REFRESH` which is `false` by default but the user can set it to true.